shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1457,12 +1457,12 @@ export default {
           this.agentConfig['cloud-provider-config'] = `secret://fleet-default:${ harvesterKubeconfigSecret?.metadata?.name }`;
 
           if (this.isCreate) {
-            this.chartValues[`${ HARVESTER_CLOUD_PROVIDER }.global.cattle.clusterName`] = this.value.metadata.name;
+            set(this.chartValues, `${ HARVESTER_CLOUD_PROVIDER }.global.cattle.clusterName`, this.value.metadata.name);
```

Comment:
We could shift the scope of the string interpolation so that it's limited to just the `HARVESTER_CLOUD_PROVIDER` key if we wanted to use assignment: 

```suggestion
            `this.chartValues[`${ HARVESTER_CLOUD_PROVIDER }`].global.cattle.clusterName = this.value.metadata.name;`
```

If I understand correctly, this would produce the same result.